src/assets/styles/baseStyles.scss

```diff
@@ -36,4 +36,22 @@ body{
     background-repeat: repeat-y;
     height: 100vh;
   }
+}
+
+@media (max-width: 768px) {
+  .light-body {
+    background-image: url('../images/background-light-mobile.png');
+    background-size: cover;
+    background-position: center;
```

Comment:
Czy to `background-position: center` jest potrzebne? Jeśli nie to proponuję nie wyrzucić z uwagi na to że, jeśli jest ten styl ustawiony to, wtedy tło nie zaczyna się od początku tak jak powinno, tylko od środka.